intrastat_base/views/account_fiscal_position_template.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2021 Akretion France (https://akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
```

Comment:
@alexis-via This is an addition. Could you maybe put it in another commit ?

Reply:
I removed this contribution. Let's focus on the important stuff and avoid nit-picking.